rozy/src/utils.rs

```diff
@@ -1,9 +1,41 @@
 use anyhow::Result;
 
+fn is_request_retryable_based_on_error(err: &reqwest::Error) -> bool {
+    if err.is_timeout() || err.is_connect() {
+        return true;
+    }
+
+    if err.is_decode() || err.is_body() || err.is_builder() || err.is_request() || err.is_redirect()
```

Comment:
There isn't great documentation for what these mean: https://docs.rs/reqwest/0.11.14/reqwest/struct.Error.html but we can adjust as needed.

Reply:
`err.is_redirect()` means that the `reqwest` library gave up after following 10 (by default) redirects